### DISCUSSION TITLE
ONVIF

### DISCUSSION THREAD

**Discussion by lancewicz** [roleoroleo/yi-hack-Allwinner-v2#787] [General]
Hi Roleoroleo,

Did you made a lot of changes in ONVIF? in the past I used this cameras with ONVIF NVR without any problems.
Now 0.3.0 firmware is not connecting event to event the ONVIF_Device_TestTool_v18.06 

How you are checking ONVIF compatyblity?

**Comment by lancewicz** [comment]
I check. Last working FW with my NVR is 0.2.6 
Can you please check how big was changes between 0.2.6 and 0.3.0?

**Comment by roleoroleo** [comment]
Onvif is a BIG problem because it's a horrible "standard" (IMHO).
I made a lot of changes from the beginning.
And between 0.2.6 and 0.2.7 I replaced the old onvif_srvd daemon with a new server I wrote.
So a big big change.

I test the onvif compatibility with the official test tool: Onvif Device Test Tool (actually version 18.06).
About events, the old onvif_srvd doesn't contain events support.
The new onvif_simple_server handles events with WS Notification interface.

Currenty I'm working hard in the onvif sources.
Maybe I already fixed your problem in the current beta version :)

Back to your problem, please try to capture the network traffic between the NVR and the cam and send it to me.
